### ISSUE TITLE
Missing browse scoped option

### ISSUE THREAD

**Issue by jecisc** [pharo-ide/Calypso#271]
In Nautilus, a really really cool option is "browse scoped" allowing to browse only a subset of the system (a package, a class...). This allows browsing senders/implementors/references... only in a part of the system, which is useful when we look for common methods. 

This should probably be implemented in Calypso before Pharo 7 release.

**Comment by dionisiydk** [comment]
It is already here for a long time. But approach is different.
Scopes are just available in any method browser. No need to press any button to have them. 
Notice "Current image" item in method browser. It is combo box to switch these scopes. Try to expand list.
Scopes are inherited from parent browser which spawn them. And it also includes items from selected methods.

Same widget is available for refactoring previews.

**Comment by dionisiydk** [comment]
So after discussion with Stef and Guille I realized what scenario is actually missing.
And now it is available in 0.11.4 (already integrated):

I added menu item «Set up scope» where you select multiple packages. This scope is propogated to other places.
Indication is done with special label «Scoped pckg» for «Packages» radio button. 
In addition this radio have extra logic when you click on it in checked state. In scoped mode it resets scope. In normal mode it sets up the scope of selected packages.
